packages/push/src/push_vm/push_io/mod.rs

```diff
@@ -0,0 +1,15 @@
+use std::{io::Write, string::FromUtf8Error};
+
+pub trait HasStdout {
+    type Stdout: Write;
+
+    fn stdout(&mut self) -> &mut Self::Stdout;
```

Comment:
Maybe add some docs to this method?

Reply:
I've added something here, but I'm not 100% convinced that this is terribly clear or useful. Thoughts?